plugin/ochttp/stats_transport.go

```diff
@@ -126,3 +126,73 @@ func (t *tracker) Close() error {
 	t.end()
 	return t.body.Close()
 }
+
+func (h *Handler) startStats(w http.ResponseWriter, r *http.Request) (http.ResponseWriter, *http.Request, func()) {
+	ctx, _ := tag.New(r.Context(),
```

Comment:
I don't think we should add these tags by default until we have a way to control propagation. It means that every outgoing gRPC call will have a HTTP host tag added to its metadata that will be propagated to the gRPC server and set there as well. This doesn't seem like an expected behavior. Instead, we should add the tags just around the Record call, not the context used on the Request.

Reply:
it's also a little strange that his method is in the _transport.go file